### ISSUE TITLE
Dll Injection is working but GAS is not being enabled

### ISSUE THREAD

**Issue by AZANG2EE** [aeventyr/LR2GAS_pub#2]
![image](https://user-images.githubusercontent.com/129039586/227864637-bc51a603-bf3e-482a-b693-bd93adc47b63.png)

![image](https://user-images.githubusercontent.com/129039586/227864751-7bccc311-f3bc-42d5-af8a-7002644b6f71.png)

![image](https://user-images.githubusercontent.com/129039586/227864759-35b47774-509f-4ba9-9ce5-d0d951a01031.png)

I have tried to inject LR2GAS.dll into LR2HBody.exe using regedit. 

As we can see in the process explorer, it seems LR2GAS.dll is being successfully injected into LR2HBody.exe, but the GAS is not working.. 

I have also used 'Remote DLL' to inject dll. All three methods (CreateRemoteThread, NtCreateThread, QueueUseAPC) showed injection was successfuly done, but gauge auto shift was not working for all three methods.

Do you have any clues to resolve this issue?

**Comment by aeventyr** [comment]
I haven't used regedit to inject dll. The Remote DLL program worked for me with either CreateRemoteThread or NTCreateThread injection method, but QueueUseAPC did not work.
Did you try turning off play options like gbattle or replay? GAS is disabled when those options are on.

**Comment by AZANG2EE** [comment]
Yes, I have tested it starting with "HARD" gauge option. It should shifted to GROOVE gauge when i miss notes below 2%, but constantly proceeds to failed results. I also tried Remote DLL, and turned off recording replays, but both of them also showed same symptom..

**Comment by jbscj** [comment]
Have you tried using other injection tools? For example this one [Autodllinjector](https://sourceforge.net/projects/autodllinjector/) 
If other tools still don't work leave me a message channel like your discord username, and I'll follow up

**Comment by aeventyr** [comment]
Can you try other injectors like [Auto DLL Injector](https://sourceforge.net/projects/autodllinjector/)? If still won't work please leave something like discord usernames and I'll follow up

**Comment by AZANG2EE** [comment]
Sorry for the late reply, I have tested Auto DLL injector and confirmed dll injection was successfully done, but still GAS ain't enabled. 

Interestingly, I have tried Auto DLL injector to inject your other plugin, LR2Unrandomizer (https://github.com/aeventyr/LR2Unrandomizer), and this dll worked fine for me. 

I have tested R-Random feature on the ★★1 Nichirin -HighSpeed edition-. It starts with stairs pattern and I could generate different r-randomed stairs pattern in multiple attempts successfully.

At first, I thought my LR2Hbody.exe have some problem with the injection, but it turned out it is not. So, I kinda feel lost why LR2gas.dll ain't working..

My discord username is AZANG_BM#3055. I'd be glad if you follow up and suggest me how to resolve this issue. Many thanks for devlopment and QA for this plugin!
